android/BOINC/app/src/main/java/edu/berkeley/boinc/adapter/PrefsListItemWrapperClasses.kt

```diff
@@ -0,0 +1,140 @@
+/*
```

Comment:
Please move classes in this file into separate one.
Generic rule is 1 class - 1 file except. We're not running out of empty space to store list of files so there is no need to put several classes into one file. Also it's easier to search for a class just looking at the name of the file.

Reply:
In Kotlin, it's encouraged to place multiple related declarations in the same file: https://kotlinlang.org/docs/reference/coding-conventions.html#source-file-organization